src/Raven.Server/Documents/DocumentsStorage.cs

```diff
@@ -1150,6 +1150,25 @@ public IEnumerable<Tombstone> GetTombstonesFrom(DocumentsOperationContext contex
             }
         }
 
+        public IEnumerable<Tombstone> GetTombstonesInReverseOrderFrom(DocumentsOperationContext context, long etag, long start, long take)
```

Comment:
Please rename to `GetTombstonesInReverseEtagOrderFrom` for consistency with other methods e.g. `GetDocumentsInReverseEtagOrder`